testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -489,6 +499,11 @@ def test_iter_as_aux_lowf(self, ref, reader):
                                          decimal=ref.prec)
 
 
+    def test_pickle_next_ts_reader(self, reader):
```

Comment:
Maybe add a comment or a link to an issue to explain why this is an important test.